set.mm

```diff
@@ -285606,6 +285662,252 @@ Additional definitions for (multivariate) polynomials
       VRRVCAWEVSWGJKBCEGIVDVEVQVIVNVHVPBQOZBMNVPSVQSWHSWDVISVFUKVG $.
   $}
 
+  ${
+    $d f h i k r x y $.
+    $( Define the differentiation operation on multivariate polynomials.
+       ` ( ( ( I mPSDer R ) `` X ) `` F ) ` is the partial derivative of the
+       polynomial ` F ` with respect to ` X ` .  (Contributed by Mario
+       Carneiro, 21-Mar-2015.) $)
+    df-psd $a |- mPSDer = ( i e. _V , r e. _V |-> ( x e. i |->
+      ( f e. ( Base ` ( i mPwSer r ) ) |->
+        ( k e. { h e. ( NN0 ^m i ) | ( `' h " NN ) e. Fin } |->
+          ( ( ( k ` x ) + 1 ) ( .g ` r ) ( f `
+            ( k oF + ( y e. i |-> if ( y = x , 1 , 0 ) ) ) ) ) ) ) ) ) $.
+  $}
+
+  ${
+    $d I f h i k r x y $.  $d R f i k r x $.  $d B i r $.  $d D i r $.
+    $d ph i r $.
+    psdffval.s $e |- S = ( I mPwSer R ) $.
+    psdffval.b $e |- B = ( Base ` S ) $.
+    psdffval.d $e |- D = { h e. ( NN0 ^m I ) | ( `' h " NN ) e. Fin } $.
+    psdffval.i $e |- ( ph -> I e. V ) $.
+    psdffval.r $e |- ( ph -> R e. W ) $.
+    $( Value of the power series differentiation operation.  (Contributed by
+       SN, 11-Apr-2025.) $)
+    psdffval $p |- ( ph -> ( I mPSDer R ) = ( x e. I |->
+      ( f e. B |->
+        ( k e. D |->
+          ( ( ( k ` x ) + 1 ) ( .g ` R ) ( f `
+            ( k oF + ( y e. I |-> if ( y = x , 1 , 0 ) ) ) ) ) ) ) ) ) $=
+      ( co cmpt vi vr cvv cv cmps cbs cfv ccnv cn cima cfn wcel cmap crab caddc
+      cn0 c1 weq cc0 cif cof cmg cpsd cmpo wceq df-psd a1i simpl oveq12 eqtr4di
+      fveq2d oveq2d rabeqdv fveq2 adantl eqidd mpteq1d oveq123d mpteq12dv elexd
+      wa mptexd ovmpod ) AUAUBKFUCUCBUAUDZHWDUBUDZUESZUFUGZJIUDUHUIUJUKULZIUPWD
+      UMSZUNZBUDJUDZUGUQUOSZWKCWDCBURUQUSUTZTZUOVAZSZHUDZUGZWEVBUGZSZTZTZTZBKHD
+      JEWLWKCKWMTZWOSZWQUGZFVBUGZSZTZTZTZVCUCVCUAUBUCUCXCVDVEABCHIUAJUBVFVGWDKV
+      EZWEFVEZWAZXCXKVEAXNBWDXBKXJXLXMVHZXNHWGXADXIXNWGGUFUGDXNWFGUFXNWFKFUESGW
+      DKWEFUEVINVJVKOVJXNJWJWTEXHXNWJWHIUPKUMSZUNEXNWHIWIXPXNWDKUPUMXOVLVMPVJXN
+      WLWLWRXFWSXGXMWSXGVEXLWEFVBVNVOXNWLVPXNWPXEWQXNWNXDWKWOXNCWDKWMXOVQVLVKVR
+      VSVSVSVOAKLQVTAFMRVTABKXJLQWBWC $.
+
+    $d X f k x y $.  $d B f x $.  $d D x $.  $d ph x $.
+    psdfval.x $e |- ( ph -> X e. I ) $.
+    $( Value of the power series differentiation operation.  (Contributed by
+       SN, 11-Apr-2025.) $)
```

Comment:
This (and the following) comment(s) are not correct and should be more specific.